addons/website/views/snippets/s_numbers_boxed.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_boxed" name="Numbers boxed">
+    <section class="s_numbers_boxed pt80 pb80 o_cc o_cc5">
+        <div class="container">
+            <div class="row o_grid_mode" style="column-gap: 48px;" data-row-count="11">
```

Comment:
Apply a vertical gap as well. This ensures that, if the user clones a column, the new column will be spaced out correctly  😉

```suggestion
            <div class="row o_grid_mode" style="gap: 48px;" data-row-count="11">
```